deploy/helm/templates/rbac/cluster_pod_required_role.yaml

```diff
@@ -0,0 +1,68 @@
+# permissions for end users to edit clusters.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubeblocks-cluster-pod-role
+  labels:
+    {{- include "kubeblocks.labels" . | nindent 4 }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+- apiGroups:
+  - "dataprotection.kubeblocks.io"
+  resources:
+  - backups/status
+  verbs:
+  - get
+  - update
+  - patch
+- apiGroups:
+  - "dataprotection.kubeblocks.io"
```

Comment:
why is `backups.dataprotection.kubeblocks.io` API access needed?

Reply:
backupservice need it, @wangyelei

Reply:
maintain this access in a different clusterrole object as `kubeblocks-cluster-pod-role` doens't quite match the basic workload pod usage